lib/Core/Algorithm/include/algorithm.h

```diff
@@ -52,6 +52,7 @@ limitations under the License.
 #define HEADING_MAG             1
 #define HEADING_GNSS_LOW        2
 #define HEADING_GNSS_HIGH       3
+#define HEADING_RTK             4
```

Comment:
Instead of outputting whether we use RTK heading or not in the status byte, I think you could output the headingIni value.